### DISCUSSION TITLE
Prevent video stream playback from being stopped / put into hibernation / discarded in hidden tabs

### DISCUSSION THREAD

**Discussion by ghost** [night/betterttv#5958] [Ideas]
Prevent video stream playback from being stopped / put into hibernation / discarded in hidden tabs.

Example: 
Two pages with drops for different games are opened, you stay on A, get all the current drops on that, B won't start since stream stopped / page got hibernated / discarded.

Extra apps could be used, but it would be much better if it was all-in-one with BTTV

Thank you for your time.

**Comment by night** [comment]
Because BetterTTV runs on the Twitch page itself, it is not able to prevent browser-level modifications that pause or otherwise hibernate pages. You would unfortunately need to leverage a third party extension to accomplish this at this time.
